cluster/packages.sls

```diff
@@ -39,3 +39,14 @@ install_cluster_packages:
       - sbd
 
 {% endif %}
+
+{% if grains['provider'] == "azure" %}
```

Comment:
We cannot use `grains['provider']` here as this entry is not standard (only available in the terraform project).
We need other way to know if the formula is being executed in Azure.
I see 2 options:
1. Get the provider from other pillar entry (I think we already have that)
2. Find if the deployment is in azure in other way
3. Install socat in all version, It doesn't look like harmful, but I would try to avoid this

Reply:
Thanks @arbulu89 I was undecided to choose between pillar/grains earlier. I have updated to retrieve provider from pilllar entry now